packages/rum/src/domain/rumEventsCollection/longTask/longTaskCollection.spec.ts

```diff
@@ -1,8 +1,8 @@
-import { setup, TestSetupBuilder } from '../../../test/specHelper'
-import { RumPerformanceEntry } from '../../browser/performanceCollection'
-import { RumEventCategory } from '../../index'
-import { RumEventType } from '../../typesV2'
-import { LifeCycleEventType } from '../lifeCycle'
+import { setup, TestSetupBuilder } from '../../../../test/specHelper'
+import { RumPerformanceEntry } from '../../../browser/performanceCollection'
+import { RumEventCategory } from '../../../index'
```

Comment:
We might want to introduce some TS path to avoid deep relative require paths

Reply:
Yes, it could be nice.
Let's see in which state we are at the end of the migration, we'll probably want to re-challenge the file organization.
It could be easier to define the right relative path.